app/Models/Post.php

```diff
@@ -64,6 +74,12 @@ public function reports()
         return $this->hasMany('App\Models\Report', 'post_id');
     }
 
+    public function reported($post_id) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Post.php</br>**Line**: 77</br>Opening brace should be on a new line